alacritty/src/window.rs

```diff
@@ -153,20 +157,23 @@ impl Window {
         event_loop: &EventLoop<E>,
         config: &Config,
         size: Option<PhysicalSize<u32>>,
-        #[cfg(not(any(target_os = "macos", windows)))] wayland_event_queue: Option<&EventQueue>,
+        #[cfg(all(feature = "wayland", not(any(target_os = "macos", windows))))]
+        wayland_event_queue: Option<&EventQueue>,
     ) -> Result<Window> {
         let window_config = &config.ui_config.window;
         let window_builder = Window::get_platform_window(&window_config.title, &window_config);
 
         // Disable vsync on Wayland.
```

Comment:
This comment isn't entirely accurate anymore. I'd go more for something like this:

```suggestion
        // Check if we're running Wayland to disable vsync.
```

That way we still have the vsync thing in here, but it doesn't act like this is directly disabling it. Though I'm open for suggestions.